### DISCUSSION TITLE
Education pack benefits not received

### DISCUSSION THREAD

**Discussion by Malware6666** [community/community#142805] [GitHub Education]
### Hi everyone,

It’s been more than 3 days still not received my benefits how long i have to wait ?

**Comment by davevad93** [comment]
Hi @Malware6666 ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @Malware6666 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @Malware6666 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

**Reply by Malware6666** [reply]
Thanks I received my benefits few hours ago ❤️

**Reply by davevad93** [reply]
You're welcome @Malware6666 , have a nice day! 👍
